src/screens/landingPageScreens/ResourcesScreen.tsx

```diff
@@ -0,0 +1,88 @@
+
+
+interface CardData {
+    title: string;
+    description: string;
+    link: string;
+    linkUrl: string;
+}
+
+interface ResourcesScreenProps {
```

Comment:
This could be moved to the utils class as above `src/data/utils` or  `src/data/utils/interfaces` and reused in the future.